Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/timeline.js

```diff
@@ -662,6 +672,32 @@ class TimelineFromEndpoint {
             this.regressions.points = this.computedRegressionPoints(allConfigResults);
         }
     }
+    getRangeUuids(centerUuid, allConfigResults) {
+        const i = allConfigResults.findIndex(r => r.uuid === centerUuid);
+        if (i < 0) return null;
+
+        const win = this.regressions.analysisWindow;
+        const start = Math.max(0, i - win);
+        const end = Math.min(allConfigResults.length - 1, i + win - 1);
+
+        let leftIdx = -1;
+        let rightIdx = -1;
+
+        for (let j = start; j <= end; j++) {
+            const { failureType } = this.getTestResultStatus(allConfigResults[j], willFilterExpected);
+            const failure = !!failureType && failureType !== 'success';
```

Comment:
From offline discussion: you should double-check to make sure that `failureType` will ever be falsy. Otherwise, the first condition of `!!failureType` will never be `false` and, thus, is unnecessary.